ironfish/src/wallet/walletdb/walletdb.ts

```diff
@@ -604,6 +660,61 @@ export class WalletDB {
       [account.prefix, [assetId, [sequence, [value, noteHash]]]],
       tx,
     )
+    await this.deleteValueToUnspentNoteHash(account, decryptedNote, tx)
+  }
+
+  async *loadUnspentNoteHashesByValue(
```

Comment:
This function says it's loading note hashes but it's not.

Should it be, `loadUnspentNotesByValue`? It's the mirror function of `loadUnspentNotes`.

Reply:
yeah that is a more consistent, i changed it.